test/basic.html

```diff
@@ -206,6 +206,77 @@
         });
       });
 
+      suite('accessibility', function() {
+        var LEFT = 37;
+        var RIGHT = 39;
+        var tabs;
+
+        setup(function () {
+          tabs = fixture('basic');
+          ensureDocumentHasFocus();
```

Comment:
Have you tried calling this method on the tests that require focus? For what i could experience, calling it on `setup` doesn't help.